jcl/j9modules.xml

```diff
@@ -51,6 +57,8 @@
 				<removeexport package="com.ibm.dataaccess" />
 				<removeexport package="com.ibm.jvm" />
 				<removeexport package="com.ibm.oti.shared" />
+				<removeexport package="com.ibm.oti.util" />
+				<removeexport package="com.ibm.oti.vm" />
```

Comment:
Again, these two packages are exported to modules other than `com.ibm.management`.

Reply:
This is to remove the export statements containing `com.ibm.management` during composing. The export statements without the old package name are added back in by `module-info.java.extra` files.